config/symbols.us.stno3.txt

```diff
@@ -15,6 +15,10 @@ g_eBreakableExplosionTypes = 0x80180E78;
 g_eBreakableanimSets = 0x80180E80;
 g_eBreakableDrawModes = 0x80180E90;
 g_TrapDoorFlag = 0x8018123C;
+EntityCastleBridgeUVs = 0x80181940;
```

Comment:
Would you mind adding `g_` to match our style? https://github.com/Xeeynamo/sotn-decomp/blob/master/docs/STYLE.md#naming

Other than that looks good!

Reply:
yeah, it was a bit unclear how to name these, whether they are truly global. Thanks for the review!